src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -520,15 +520,17 @@ private void regenerateCitationKeys(List<ParserResult> loaded) {
      * @return A parser result containing the entries fetched or null if an error occurred.
      */
     private Optional<ParserResult> fetch(String fetchCommand) {
-        if ((fetchCommand == null) || !fetchCommand.contains(":")) {
-            System.out.println(Localization.lang("Expected syntax for --fetch='<name of fetcher>:<query>'"));
+        if ((fetchCommand == null) || !fetchCommand.contains(":") ||
+                !(fetchCommand.toLowerCase().endsWith(":bibtex") || fetchCommand.toLowerCase().endsWith(":biblatex"))) {
+            System.out.println(Localization.lang("Expected syntax for --fetch='<name of fetcher>:<query>:<bib entry format>'"));
```

Comment:
If these changes are approved this localized text has to be adapted

Reply:
Due to the decision in ADR-0012, this hole change is obsolete? --> `git checkout upstream/master -- src/main/java/org/jabref/cli/ArgumentProcessor.java` (to revert the changes in this file)

(If not, there should be a default mode, if possible, the mode of the bibdatabase should be used)